app/src/main/java/com/xmartlabs/template/extensions/ui/ViewPagerExtensions.kt

```diff
@@ -0,0 +1,20 @@
+@file:Suppress("NOTHING_TO_INLINE")
```

Comment:
why not inlining?

Reply:
we use `inline fun` after that and the linter shows a warning with the methods if the suppression is not define